airflow/www/static/ansi_up.min.js

```diff
@@ -0,0 +1,7 @@
+//# sourceMappingURL=/sm/e54fac82455c5244f73ba6a437a8ca0fefeba655e9f1fae1535a2c5dad960c25.map
```

Comment:
Since we've got a source map line in this file we should also include the ansi_up.js.map file for this release.

Reply:
Where did you get this file from btw? And what version of ansi_up does it related to?

Reply:
This is the latest version (4.0.4).
I generated this minified version by myself from the source ansi_up.js file.
because I couldn't find the minified version in the ansi_up repo